src/gc/impl/conservative/gc.d

```diff
@@ -243,6 +247,39 @@ debug (LOGGING)
 
 /* ============================ GC =============================== */
 
+
+debug(PRINTF)
+void printGCBits(GCBits* bits)
```

Comment:
I think these and other debug functions (LOGGING/PRINTF) should go to the bottom of the file, but that's for another PR.